index.html

```diff
@@ -81,11 +81,105 @@ <h1>Skills</h1>
                         <img src="IMG1/twitter.png" class="twitter" alt="">
                         <img src="IMG1/m.png" class="m" alt="">
                     </div>
-                    <div class="work-image">
-                        <img src="IMG1/work-element.png" alt="">
-                    </div>
                 </div>
         </section>
+
+        <section>
```

Comment:
- [ ] Kindly fix spacing between the two sections, check screenshot below ⬇️ 



Your design             |  Figma Design
:-------------------------:|:-------------------------:
![mmmesec](https://user-images.githubusercontent.com/69638013/154811329-a7f207ec-b6f0-45c8-875b-396090891c72.PNG)  | ![maemee](https://user-images.githubusercontent.com/69638013/154811308-9a6827ae-81d6-476a-8377-4469b4f5c2bf.PNG)

### OPTIONAL
- [ ] I advise you to add work close to the arrow down  as text and not an icon, this makes your website   easier to read by screen readers 🤝